src/runtime/database.cpp

```diff
@@ -34,7 +34,7 @@
 #include "status.h"
 
 // Increment every time the database schema changes
-#define SCHEMA_VERSION "5"
+#define SCHEMA_VERSION "6"
```

Comment:
ah it was https://github.com/sifive/wake/blob/master/tools/wake/main.cpp#L385

Reply:
Yeah it will error out and require wake.db to be deleted. This change isn't actually incompatible though so maybe we should leave it at 5?

Reply:
Yeah so if a wake that creates the current version uses a current database it will just ignore the unhashed files like it does today. If a wake with this version uses an old database it will just not know about some of the unhashed files but it will start adding any new ones.

Reply:
Also that line checks a different thing I think, the check for version of the database happens here: https://github.com/sifive/wake/blob/master/src/runtime/database.cpp#L163